charts/operator/README.md

```diff
@@ -0,0 +1,118 @@
+# Rookout
+
+[Rookout](http://rookout.com/) gets data from your live code, as it runs. Extract any piece of data from your code and pipeline it anywhere, in realtime, even if you’d never thought about it beforehand or created any instrumentation to collect it.
+
+## Introduction
+
+This chart installs [Rookout's k8s Operator](https://docs.rookout.com/docs/k8s-operator-setup.html) on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.9+ with Beta APIs enabled
+
+## Installing the Chart using helm
+
+To install the chart with the release name `my-release`:
```

Comment:
I think its better to change to to `rookout` so people wouldn't mistakingly run the `helm install my-release`